react/features/lobby/middleware.ts

```diff
@@ -245,24 +202,40 @@ function _handleLobbyNotification(store: IStore) {
             dispatch(hideNotification(LOBBY_NOTIFICATION_ID));
             dispatch(rejectKnockingParticipant(firstParticipant.id));
         }) ];
-    } else {
+
+        // This checks if lobby chat button is available
+        // and, if so, it adds it to the customActionNameKey array
+        if (showChat) {
+            customActionNameKey.splice(1, 0, 'lobby.chat');
+            customActionHandler.splice(1, 0, () => batch(() => {
+                dispatch(handleLobbyChatInitialized(firstParticipant.id));
+                // @ts-ignore
+                dispatch(openChat(disablePolls));
+            }));
+        }
+    }
+
+    if (knockingParticipants.length > 1) {
         descriptionKey = 'notify.participantsWantToJoin';
         notificationTitle = i18n.t('notify.waitingParticipants', {
             waitingParticipants: knockingParticipants.length
         });
         icon = NOTIFICATION_ICON.PARTICIPANTS;
+        concatText = true;
```

Comment:
We should drop `concatText` now, since it was a workaround for the mobile notifications we used to have, since they didn't have the context of title and description.